src/components/Admin/StoriesForApproval.js

```diff
@@ -0,0 +1,154 @@
+import React, { Component } from 'react';
+import { Spinner, Card, Media, Badge, Row } from 'reactstrap';
+import _ from 'lodash';
+import '../../styles/TrendingStories.css';
+import axios from 'axios/index';
+import { TRUTHTREE_URI } from '../../constants';
+import { connect } from 'react-redux';
+import ReactHtmlParser from 'react-html-parser';
+
+class StoriesForApproval extends Component {
+  constructor(props) {
+    super(props);
+    this.getStoryDetails = this.getStoryDetails.bind(this);
+    this.selectStory = this.selectStory.bind(this);
+    this.state = {
+      InitialData: [],
+      bgColor: []
+    };
+  }
+
+  contentHtml(data) {
+    return ReactHtmlParser(data);
+  }
+
+  componentDidMount() {
+    //List of stories to be approved if admin
+    axios
+      .get(`${TRUTHTREE_URI}/api/stories/pending`)
+      //Change the api call to unapproved stories
+      .then(response => {
+        let color = [];
+        for (var i = 0; i < response.data.length; i++) {
```

Comment:
Why do we need a separate array for colors? Can we do sth like this:
```
response.data.each(story => {
  story.color = 'white';
}
```

Reply:
Sure, I have created a task for myself - will work on doing this (CS-226)